src/convertToObject.js

```diff
@@ -6,7 +6,26 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
-}
+  let strToObject = sourceString
+    .trim()
+    .split(';')
```

Comment:
The 'split' method does not mutate the original string, so there's no need to make a copy of 'sourceString' before splitting it.